sycl/source/detail/queue_impl.cpp

```diff
@@ -488,9 +500,7 @@ void *queue_impl::instrumentationProlog(const detail::code_location &CodeLoc,
   if (WaitEvent) {
     device D = get_device();
     std::string DevStr;
-    if (getSyclObjImpl(D)->is_host())
-      DevStr = "HOST";
-    else if (D.is_cpu())
+    if (D.is_cpu())
```

Comment:
Not related to this PR, but this kind of conversion is duplicated in a few places (e.g., commands.cpp), we should add a common helper method for this.